### PR TITLE
Implementation after design changes

### PR DESCRIPTION
**Pull request by elenasolomon**
Implementation for:
- data container
- data block blob
- append data blob

The README was not updated.

### CODE REVIEW

**Comment by elenasolomon** [comment]
I managed to repair the tests for authentication with SAS, but they are still failing because the taskcluster-client is not updated with the new Auth API (azureBlobSAS). I will soon make a new PR for this.

File: README.md [hunk 1]
```diff
@@ -1,86 +1,230 @@
...
 
 ```js
-// create an instance of Container without specifing the JSON schema
-var container = blobStorage.createContainer({
-    name: containerName,
-});
+    await container.remove('state-blob', true);
+```
+
+### DataBlocKBlob and AppendDataBlob
```
Comment (jhford) [inline README.md]: I think you have an extra K in there :)

File: README.md [hunk 2]
```diff
@@ -1,86 +1,230 @@
...
+
+DataBlockBlob is a wrapper over an Azure block blob which stores a JSON data which is conform with schema defined at container
```
Comment (jhford) [inline README.md]: "...stores JSON data that conforms to the schema defined..."

**Review by jhford — APPROVED** [review]
This looks great to me, let's merge it.  We'll find out if there's issues when we start working on the provisioner :)

**Pull request merged**